pinot-core/src/test/java/org/apache/pinot/core/data/manager/BaseTableDataManagerTest.java

```diff
@@ -191,7 +191,6 @@ public void testReloadSegmentUseLocalCopy()
       fail();
     } catch (Exception e) {
       // As expected, segment reloading fails due to missing the local segment dir.
-      assertTrue(e.getMessage().contains("does not exist or is not a directory"));
```

Comment:
did reloadSegment() complete w/o any errors? that's a bit unexpected as L188 deleted the local seg dir. same question for next test case.

Reply:
Still fails with an exception (If it didn't, test would fail (see https://github.com/apache/pinot/pull/10089/files#diff-2ca49ed375e7b21a34397df398e1586e8ee64283461abe4a9a2cae17097fbab9R191))
Just not the one with the message `does not exist or is not a directory`...

Reply:
Same for next one